sphinx/source/docs/gallery.json

```diff
@@ -3,277 +3,277 @@
     },
     {
       "path": "examples/plotting/file/bar_nested.py",
       "name": "bar_nested",
-      "alt": "Thumbnail link to the examples/plotting/file/bar_nested.py example"
+      "alt": "Thumbnail link to the examples/plotting/file/bar_nested.py example shows a grouped bar chart on a hierachical categorical axis of years for different categories of fruit. Each group has three blue bars corresponding to random values for years 2015, 2016 and 2017."
     },
     {
       "path": "examples/plotting/file/candlestick.py",
       "name": "candlestick",
-      "alt": "Thumbnail link to the examples/plotting/file/candlestick.py example"
+      "alt": "Thumbnail link to the examples/plotting/file/candlestick.py example shows a candlestick plot of historical Microsoft stock prices."
     },
     {
       "path": "examples/plotting/file/legend.py",
       "name": "legend",
-      "alt": "Thumbnail link to the examples/plotting/file/legend.py example"
+      "alt": "Thumbnail link to the examples/plotting/file/legend.py example shows a row of two plots that have sine waves in different scatter and line styles, illustrated in accompanying legends."
     },
     {
       "path": "examples/plotting/file/bar_colors.py",
       "name": "bar_colors",
-      "alt": "Thumbnail link to the examples/plotting/file/bar_colors.py example"
+      "alt": "Thumbnail link to the examples/plotting/file/bar_colors.py example shows a simple bar chart of six bars shaded according to the Bright6 palette."
     },
     {
       "path": "examples/plotting/file/box_annotation.py",
       "name": "box_annotation",
-      "alt": "Thumbnail link to the examples/plotting/file/box_annotation.py example"
+      "alt": "Thumbnail link to the examples/plotting/file/box_annotation.py example shows a timeseries plot of glucose data readings with red box annotations marking regions where the readings are too high or too low."
     },
     {
       "path": "examples/plotting/file/les_mis.py",
       "name": "les_mis",
-      "alt": "Thumbnail link to the examples/plotting/file/les_mis.py example"
+      "alt": "Thumbnail link to the examples/plotting/file/les_mis.py example shows a reproduction of D3’s Les Misérables Co-occurrence chart that illustrates which characters were in scenes together with a shaded square in the matrix plot."
     },
     {
       "path": "examples/plotting/file/bar_dodged.py",
       "name": "bar_dodged",
-      "alt": "Thumbnail link to the examples/plotting/file/bar_dodged.py example"
+      "alt": "Thumbnail link to the examples/plotting/file/bar_dodged.py example shows a grouped bar chart using a visual dodge on the bars, rather than a hierarchical axis. Each group has a silver, blue, and ed bar illustrated in the accompanying legend."
```

Comment:
```suggestion
      "alt": "Thumbnail link to the examples/plotting/file/bar_dodged.py example shows a grouped bar chart using a visual dodge on the bars, rather than a hierarchical axis. Each group has a silver, blue, and red bar illustrated in the accompanying legend."
```